service/spi/src/main/java/gov/va/vro/service/spi/model/AssessmentResult.java

```diff
@@ -0,0 +1,37 @@
+package gov.va.vro.service.spi.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Map;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class AssessmentResult {
```

Comment:
This class confuses me. I gather we return this after assessment. But the controller expects the object that healthassessment microservice uses. If this is indeed changing the message it should not really to my understanding. Save to dbs were supposed to be audit logs that do not change anything in the messages. I actually do not understand if this really becomes the return value and how it is used. I need to dig in some more.